src/generic/stage1/utf8_lookup3_algorithm.h

```diff
@@ -205,19 +205,27 @@ namespace utf8_validation {
       this->error |= this->prev_incomplete;
     }
 
-    really_inline void check_next_input(simd8x64<uint8_t> input) {
-      if (likely(is_ascii(input))) {
+    really_inline void check_next_input(const simd8x64<uint8_t>& input) {
+      if (unlikely(!is_ascii(input))) {
```

Comment:
Why did you switch the likely and unlikely case? I don't expect it to make a difference either way, just curious.

Reply:
Hacking. I undid it.